src/Console/Command/FixCommand.php

```diff
@@ -506,6 +434,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @param string          $format
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return ReportInterface
+     */
+    protected function createReport($format, InputInterface $input, OutputInterface $output)
+    {
+        $reportClass = self::$reports[$format];
+
+        /** @var ReportInterface $report */
+        $report = new $reportClass();
+        $report->setDecoratedOutput($output->isDecorated());
+        $report->setShowAppliedFixers(OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity());
+        $report->setShowDiff((bool) $input->getOption('diff'));
```

Comment:
Please remove that and in places when you will decide to show diff or not use `if (!empty($fixResult['diff'])) {`